doc/rust.txt

```diff
@@ -95,6 +95,19 @@ g:cargo_manifest_name~
 	    let g:cargo_manifest_name = 'Cargo.toml'
 <
 
+                                                          *g:rust_playpen_url*
+g:rust_playpen_url~
+	Set this option to override the url for the playpen to use:
+	    let g:cargo_manifest_name = 'Cargo.toml' >
```

Comment:
Copypasta oversight—this example is not for the config var being documented here :grin: